gremlin-dotnet/src/Gremlin.Net/Process/Traversal/Cardinality.cs

```diff
@@ -26,11 +26,16 @@ namespace Gremlin.Net.Process.Traversal
 {
 #pragma warning disable 1591
 
-    public enum Cardinality
+    public class Cardinality : EnumWrapper
     {
-        List,
-		Set,
-		Single
+        private Cardinality(string enumValue)
+            : base("Cardinality", enumValue)
+        {            
+        }
+
+        public static Cardinality List => new Cardinality("list");
+		public static Cardinality Set => new Cardinality("set");
```

Comment:
`"\n\t\t"` creates this weird layout as we use 4 spaces for indentation instead of tabs.
We can use either `"\n", followed by 8 spaces` or avoid invoking `join()` and call `.each()` on the template itself.

Reply:
I decided to go with the latter so that the template can handle the formatting itself.